docs/src/pages/reference/data_to_md.ts

```diff
@@ -120,19 +136,17 @@ const displayVariant = (variant: EnumVariant, item: EnumInfo): string => {
         entry.endsWith(`/${variant.field_type}.json`)
       );
       if (innerType) {
-        let innerData: StructInfo | EnumInfo = JSON.parse(
-          readFileSync(`public/autodoc/${innerType}`).toString()
-        ).item;
-        content += `\n\n${briefItem(innerData)}`;
+        let data = JSON.parse(readFileSync(`public/autodoc/${innerType}`).toString());
+        content += `\n\n${briefItem(data.item, data.name)}`;
       }
     }
   } else if (variant.type == VariantType.Struct) {
     content += '\n\n|Field|Type|Description|\n|---|---|---|';
     if (item.tag) {
-      content += `\n|${item.tag}|"${switchCase(
-        variant.name,
-        item.rename_all
-      )}"|The tag of this variant|`;
+      content += `\n|${item.tag}|"${switchCase(variant.name, item.rename_all)}|${getTagDescription(
```

Comment:
What am I meant to do about it?

Reply:
possibly pre-assign values to reduce the line length, shouldn't matter since it's server-side anyway

(I don't know how much optimisations node-powered TS code has, I do sure hope it's at-least half as good as LLVM)